mithril-client-wasm/www/index.js

```diff
@@ -64,9 +64,26 @@ function format_tx_list(transactions_hashes) {
   return "<ul>" + transactions_hashes.map((tx) => "<li>" + tx + "</li>").join("") + "</ul>";
 }
 
+function client_options_with_custom_headers() {
+  // The following header is set as an example.
+  // It's used to demonstrate how to add headers.
+  let http_headers_map = new Map();
+  http_headers_map.set("Content-Type", "application/json");
+
+  // The headers below are examples of headers that might be used when interacting with a proxy.
+  // They are commented out because aggregators don't allow these headers.
+  // http_headers.set("Authorization", "Bearer YourBearerToken");
+  // http_headers.set("X-My-Custom-Header", "YourCustomHeaderValue");
```

Comment:
Maybe this can be removed as the message is a bit confusing IMO

Reply:
Done ✅